app/app.go

```diff
@@ -84,6 +85,7 @@ type App struct {
 	AlertStore    alert.Store
 	AlertLogStore alertlog.Store
 
+	BasicStore            *basic.Store
```

Comment:
Maybe call the prop `AuthBasicStore` so it hints a bit more about what it is when being referenced.